src/bin/ion/commands/generate/model.rs

```diff
@@ -496,6 +491,7 @@ mod model_tests {
                         ),
                     ]
                     .into_iter(),
+                    true,
```

Comment:
Same thing- what's the `true`?

Reply:
This is part of the code gen model. It is something to do with whether the fields are closed for a given type. I'll create an issue to replace it with an enum.